storage/gcloud/aio/storage/blob.py

```diff
@@ -120,12 +132,30 @@ async def get_signed_url(  # pylint: disable=too-many-locals
 
         str_to_sign = '\n'.join(['GOOG4-RSA-SHA256', request_timestamp,
                                  credential_scope, canonical_req_hash])
-        signed_resp = await iam_client.sign_blob(
-            str_to_sign, service_account_email=service_account_email,
-            session=session)
 
-        signature = binascii.hexlify(
-            decode(signed_resp['signedBlob'])).decode()
+        marker_id, key_bytes = pem.readPemBlocksFromFile(
+            io.StringIO(key), PKCS1_MARKER, PKCS8_MARKER)
+        # Given `pem.readPemBlocksFromFile(key, *args)`, returns `marker_id`
+        # as the index of the args which matched or -1 for an invalid/unmatched
+        # key.
+        if marker_id == -1:
```

Comment:
nit: What does `-1` (and `1` below) correspond with? If it means `PKCS1_MARKER_INVALID` or something something specific, a username might provide insight into what `marker_id` indicates.

Reply:
Was the comment the line above this unclear? How can I improve it?

Basically, the function I'm calling works very roughly like:
```python
def readPemBlocksFromFile(key, *args):
    marker_id = -1
    for i, arg in args:
        if matches_type(key, arg):
            marker_id = i
            break
    return marker_id
```

Reply:
I somehow missed the comment. That is clear. I take back my `nit`.

I still think there is value with having a clear variable name (such as `marker_id == NO_MATCH` or `INVALID_KEY`). But the comment is very clear.